Automattic/crowdsignal-forms
# Prepare v1.5.7 release

This PR updates the needed files to release v1.5.7

## Test instructions
Checkout and `make client`. Check the changelog entries are in effect:

* Fix toolbar remaining behind the feedback overlay (#170)
* Make video iframe dimension-less, add CSS rules for it (#169)
* Fix new interface package compatibility (#168)
* Fix little typo on the welcome screen (#167)
